src/applications/personalization/dashboard/tests/helpers.unit.spec.js

```diff
@@ -99,7 +100,7 @@ describe('profile helpers:', () => {
         sipEnabledConfig => sipEnabledConfig.formId,
       );
       sipEnabledFormIds.push('complaint-tool');
-      expect(allFormIds).to.deep.equal(allMappedIds);
+      expect(new Set(allFormIds)).to.deep.equal(new Set(allMappedIds));
```

Comment:
seems silly that these two arrays need to be in the exact same order, so I'm making them Sets for comparison purposes. I can't foresee these arrays ever needing to have duplicates so this should be safe moving forward.